site/content/usage/13-iamserviceaccounts.md

```diff
@@ -6,27 +6,27 @@ url: usage/iamserviceaccounts
 
 ## Introduction
 
-Amazon EKS supports IAM Roles for Service Accounts that allows cluster operators to map AWS IAM Roles to Kubernetes Service Accounts (IRSA).
+Amazon EKS supports IAM Roles for Service Accounts (IRSA) that allows cluster operators to map AWS IAM Roles to Kubernetes Service Accounts.
 
-This provides fine-grain permission management for apps that run on EKS and use other AWS services. These could be apps that use S3,
-any other data services (RDS, MQ, STS, DynamoDB), or a Kubernetes components like ALB Ingress controller or External DNS.
+This provides fine-grained permission management for apps that run on EKS and use other AWS services. These could be apps that use S3,
+any other data services (RDS, MQ, STS, DynamoDB), or Kubernetes components like AWS ALB Ingress controller or ExternalDNS.
 
 You can easily create IAM Role and Service Account pairs with `eksctl`.
 
 <!-- TODO: links to official docs -->
```

Comment:
Could you please add a link in under 'IAM Roles for Service Accounts (IRSA)' in the first paragraph?